content/blog/get-started-with-docker-on-aws-fargate-using-pulumi/index.md

```diff
@@ -89,7 +89,7 @@ export const url = web.endpoint.hostname;
 ## Step 1. Create a Cluster
 
 The opening stanza imports Pulumi's
-[open source AWSX NPM package]({{< relref "/docs/reference/pkg/nodejs/pulumi/awsx" >}}), `@pulumi/awsx`. It
+[open source AWSX NPM package]({{< relref "/docs/guides/crosswalk/aws/" >}}), `@pulumi/awsx`. It
```

Comment:
I think this should be reverted as it is talking about the `awsx` npm package specifically.